connection.js

```diff
@@ -37,7 +37,8 @@ class Connection {
         });
 
         this.connected_games = {};
-        this.games_by_player = {};     // Keep track of active games per player
+        this.connected_game_timeouts = {};
```

Comment:
I think you need to do a git pull off of devel. `this.connected_game_timeouts = {};` was removed in a merged PR, it shouldn't be getting merged back in.

Reply:
i pulled upstream devel and manually resolved merge conflict in https://github.com/online-go/gtp2ogs/pull/124/commits/42293545ca93fb513ed998026bd2366fd508f8cd @roy7

Reply:
Ok just remove that line from your copy then please. There is no more connected_game_timeouts array in current /devel.

Reply:
i'm still unsure if i pulled correctly

so i'd rather have a look at it next time @roy7 

maybe its not the only difference with latest devel